pkg/metrics/api/api.go

```diff
@@ -0,0 +1,456 @@
+package api
+
+import (
+	"math"
+	"net/http"
+	"os"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/openshift/installer/pkg/metrics/builder"
+	"github.com/openshift/installer/pkg/metrics/pushclient"
+)
+
+var (
+	// INVOCATION METRIC
+
+	// ClusterInstallationCreateJobName is the variable that stores the metric name for the create cluster details.
```

Comment:
I don't think these comments are helpful and IMHO make the code harder to read.

Reply:
They have to be public so the installer can use them and golang expects documentation for public fields. They are just names so I don't think we can give any useful info in the comments. Let me know if there is something better we can do here.